client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx

```diff
@@ -43,129 +55,138 @@ const SitesDataViews = ( {
-				if ( isLoading ) {
-					return <TextPlaceholder />;
-				}
-				return (
-					<span className="sites-dataviews__favorite-btn-wrapper">
-						<SiteSetFavorite
-							isFavorite={ item.isFavorite || false }
-							siteId={ item.site.value.blog_id }
-							siteUrl={ item.site.value.url }
+			{
+				id: 'site',
+				header: translate( 'SITE' ),
+				getValue: ( { item }: { item: SiteData } ) => item.site.value.url,
+				render: ( { item }: { item: SiteData } ) => {
+					if ( isLoading ) {
+						return <TextPlaceholder />;
+					}
+					const site = item.site.value;
+					return (
+						<SiteDataField
+							site={ site }
+							isLoading={ isLoading }
+							onSiteTitleClick={ openSitePreviewPane }
 						/>
-					</span>
-				);
+					);
+				},
+				enableHiding: false,
+				enableSorting: true,
 			},
-			enableHiding: false,
-			enableSorting: false,
-		},
-		{
-			id: 'actions',
-			header: translate( 'ACTIONS' ),
-			getValue: ( { item }: { item: SiteData } ) => item.isFavorite,
-			render: ( { item }: { item: SiteData } ) => {
-				if ( isLoading ) {
-					return <TextPlaceholder />;
-				}
-				return <SiteActions isLargeScreen site={ item.site } siteError={ item.site.error } />;
+			{
+				id: 'stats',
+				header: 'STATS',
+				getValue: () => 'Stats status',
+				render: ( { item }: { item: SiteData } ) => renderField( 'stats', item ),
+				enableHiding: false,
+				enableSorting: false,
 			},
-			enableHiding: false,
-			enableSorting: false,
-		},
-	];
-
-	// TODO: remove this hardcoded style. If we set background in the CSS it will be loaded with and without the feature flag:
-	document.body.style.backgroundColor = 'white';
+			{
+				id: 'boost',
+				header: 'BOOST',
+				getValue: ( { item }: { item: SiteData } ) => item.boost.status,
+				render: ( { item }: { item: SiteData } ) => renderField( 'boost', item ),
+				enableHiding: false,
+				enableSorting: false,
+			},
+			{
+				id: 'backup',
+				header: 'BACKUP',
+				getValue: () => 'Backup status',
+				render: ( { item }: { item: SiteData } ) => renderField( 'backup', item ),
+				enableHiding: false,
+				enableSorting: false,
+			},
+			{
+				id: 'monitor',
+				header: 'MONITOR',
+				getValue: () => 'Monitor status',
+				render: ( { item }: { item: SiteData } ) => renderField( 'monitor', item ),
+				enableHiding: false,
+				enableSorting: false,
+			},
+			{
+				id: 'scan',
+				header: 'SCAN',
+				getValue: () => 'Scan status',
+				render: ( { item }: { item: SiteData } ) => renderField( 'scan', item ),
+				enableHiding: false,
+				enableSorting: false,
+			},
+			{
+				id: 'plugins',
+				header: 'PLUGINS',
+				getValue: () => 'Plugins status',
+				render: ( { item }: { item: SiteData } ) => renderField( 'plugin', item ),
+				enableHiding: false,
+				enableSorting: false,
+			},
+			{
+				id: 'favorite',
+				//header: <Icon className="site-table__favorite-icon" size={ 24 } icon={ starFilled } />,
```

Comment:
Why is this and the Actions header commented out?

Reply:
I forgot to uncomment it. I checked how columns look without those headers (favorites and actions),  and then asked designers about it.

![image](https://github.com/Automattic/wp-calypso/assets/9832440/d96fa4f1-5934-4c4d-b883-b5ff22a04a0f)

**VS**

![image](https://github.com/Automattic/wp-calypso/assets/9832440/0c84b752-5b92-4f46-aec8-f84f46f85f6e)